### PR TITLE
Add persistent storage

### PR DESCRIPTION
**Pull request by holounic**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -1,32 +1,66 @@
...
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
```
Comment (krasimirgg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: I believe that the imports should be re-sorted [by ASCII sort order](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing).
Specifically, all import lines starting with `com.google` should come first, followed by all lines starting with `java.`, followed by all lines starting with `javax.`.

**Review by krasimirgg — CHANGES_REQUESTED** [review]
Please try and check if the comments functionality works.

When I try it, I load the page initially, I click the `Submit` button, then I click the `Get comment` button, then the page reloads with the text `undefined` below. I believe it should show the comments. Screenshot attached.
CyqzZxGxafb

**Comment by holounic** [comment]
> Please try and check if the comments functionality works.
> 
> When I try it, I load the page initially, I click the `Submit` button, then I click the `Get comment` button, then the page reloads with the text `undefined` below. I believe it should show the comments. Screenshot attached.
> CyqzZxGxafb

Sorry, this was because of wrongly resolved conflicts. Now it should be fine.

**Review by krasimirgg — APPROVED** [review]
way to go! LGTM

**Pull request merged**